editor/connections_dialog.cpp

```diff
@@ -1116,6 +1159,12 @@ void ConnectionsDock::update_tree() {
 				connection_item->set_text(0, path);
 				connection_item->set_metadata(0, connection);
 				connection_item->set_icon(0, get_theme_icon(SNAME("Slot"), SNAME("EditorIcons")));
+
+				if (_is_connection_inherited(connection)) {
+					// The Scene does not inherit this connection.
```

Comment:
```suggestion
					// The scene does not inherit this connection.
```

Reply:
Upon a second look that comment is incorrect, it's the exact opposite.